docs/Docker-install-windows.md

```diff
@@ -0,0 +1,206 @@
+
+1. Check your graphics card
+
+    It is highly recommended to use a graphics card to run fAIr. It might not work with CPU only. (You can set up and test from bottom of this document). ***TODO*** What does this mean? NVIDIA graphics cards are tested. The bottom of this document is about a tile server, whatever that is. But it is not a GPU test?! ***TODO*** Does this mean that it works for NVIDIA cards but not for cards from other brands? 
+
+    For the local fAIr installation to work the necessary drivers for the graphics card need to be installed.
+
+    By the following command you can see your graphics card and the graphics driver details and the NVIDIA container toolkit that is installed. There is more information on the [NVIDIA site](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html).
+
+    ```
+    nvidia-smi
+    ```
+
+    You need to see your graphics card details here. ***TODO*** Does this mean that if this command shows details about my GPU, then it is OK for fAIr?
+
+2. Install Docker Compose
+
+    If Docker is not installed, install it from the [Docker site](https://docs.docker.com/engine/install/).
+
+To check whether Docker is installed, type 
+
+    ```
+    docker compose version
+    ```
+
+3. Clone the repository for fAIr.
+
+    In a Windows Command Prompt window, type 
+
+    ```
+    git clone https://github.com/hotosm/fAIr.git
+    ```
+
+    This creates directory ```./fAIr```.
+
+4. Clone the Base Model and create environment variable RAMP_HOME
+
+    - Clone the RAMP code. 
+
+        ```
+        git clone https://github.com/kshitijrajsharma/ramp-code-fAIr.git 
+        ```
+
+    This creates directory ```./ramp-code-fAIr```.
+
+    - Create environment variable RAMP_HOME 
+
+        Set RAMP_HOME to be the file path to the directory ```ramp-code-fAIr```. 
+
+        ```
+        set RAMP_HOME=C:\Users\kshitij\fAIr_install\ramp-code-fAIr
+        ```
+
+    - Download pre-trained variable values for the RAMP model from [here](https://drive.google.com/file/d/1YQsY61S_rGfJ_f6kLQq4ouYE2l3iRe1k/view). 
+
+        If that doesn't work, you can alternatively use the variables in the original RAMP model, [RAMP Baseline](https://github.com/radiantearth/model_ramp_baseline/tree/main/data/input/checkpoint.tf)
+
+    - Create a new folder called ```ramp-variables```.
+
+        ```
+        mkdir ramp-variables
+        ```
+
+    - Unzip the downloaded base model variables in subdirectory ```ramp-variables```.
+
+        Move the compressed file to directory ```ramp-variables```. Right-click on it in File Explorer and choose "Extract All...". 
+
+    - Create environment variable TRAINING_WORKSPACE
+
+        Training workspace is the directory where fAIr will store its training files 
+        for example. ***TODO*** Is this about a new checkpoint after fine-tuning? The name implies that this is for temporary files.
```

Comment:
This is about a base folder where all information about training will be saved including checkpoints , It will also save preprocessed data & labels.geojson